use-cases/using-vpp-and-kata.md

```diff
@@ -0,0 +1,76 @@
+# Setup to run VPP
```

Comment:
Pet peeve - could you consider adding a table of contents here? That would make it much clearer which level each section is part of.